e2e/test/Doughnuts.test.ts

```diff
@@ -1516,4 +1518,143 @@ describe("Doughnuts", () => {
+
+    const module = [
+      {
+        name: "System",
+        block_cooldown: 0,
+        methods: [
+          {
+            name: "remark",
+            block_cooldown: 0,
+            constraints: [...pactEncoded],
+          },
+        ],
+      },
+    ];
+
+    const trnnut = new TRNNut(module);
+
+    // Add to trn domain
+    doughnut.addDomain(TRN_PERMISSION_DOMAIN, trnnut.encode());
+
+    // Sign the doughnut
+    const aliceWallet = await new Wallet(ALICE_PRIVATE_KEY);
+    const ethHash = blake2AsHex(doughnut.payload());
+    const ethSlice = Buffer.from(ethHash.slice(2), "hex");
+    const issuerSig = await aliceWallet.signMessage(ethSlice);
+    const sigUint8 = Buffer.from(issuerSig.slice(2), "hex");
+    doughnut.addSignature(sigUint8, SignatureVersion.EIP191);
+
+    // Verify that the doughnut is valid
+    const verified = doughnut.verify(holderPubkey, 5);
+    expect(verified).to.be.equal(true);
+
+    // Encode the doughnut
+    const encodedDoughnut = doughnut.encode();
+    const doughnutHex = u8aToHex(encodedDoughnut);
+
+    // try remark with maintenance mode pallet blocked
+    {
+      await finalizeTx(alith, api.tx.sudo.sudo(api.tx.maintenanceMode.blockPallet("System", true)));
+
+      const call = api.tx.system.remark(stringConstraint);
+      // Create a call with empty signature to be signed by the holder
+      nonce = ((await api.query.system.account(holder.address)).toJSON() as any)?.nonce;
+      const tx = await api.tx.doughnut.transact(call, doughnutHex, nonce, tip, "");
+      // Convert tx to u8Array and remove the first 2 bytes (Not sure why. It's to do with length)
+      const txU8a = tx.toU8a(true).slice(2);
+      const txHex = u8aToHex(txU8a);
+      const holderWallet = await new Wallet(holderPrivateKey);
+      const txHash = blake2AsHex(txHex);
+      const txSlice = Buffer.from(txHash.slice(2), "hex");
+      const holderSig = await holderWallet.signMessage(txSlice);
+
+      // Execute the transact call with.send
+      const dispatchError = await new Promise<DispatchError>((resolve, _reject) => {
+        api.tx.doughnut
+            .transact(call, doughnutHex, nonce, tip, holderSig)
+            .send((result) => {
+              const { status, dispatchError } =
+                  result as SubmittableResultValue;
+              if (!status.isFinalized) return;
+              if (dispatchError === undefined) return;
+              resolve(dispatchError);
+            })
+      });
+      const { section, name } = dispatchError.registry.findMetaError(
+          dispatchError.asModule
+      );
+
+      expect(section).to.equal("system");
+      expect(name).to.equal("CallFiltered");
+    }
+
+    // try remark with maintenance mode call blocked
+    {
+      await finalizeTx(alith, api.tx.sudo.sudo(api.tx.maintenanceMode.blockPallet("System", false)));
+      await finalizeTx(alith, api.tx.sudo.sudo(api.tx.maintenanceMode.blockCall("System", "remark", true)));
```

Comment:
Improve CI time (~4s atleast)
```suggestion
      await finalizeTx(alith, api.tx.utility.batch([
        api.tx.sudo.sudo(api.tx.maintenanceMode.blockPallet("System", false)),
        api.tx.sudo.sudo(api.tx.maintenanceMode.blockCall("System", "remark", true))
      ]));
```